physical/cockroachdb/cockroachdb_test.go

```diff
@@ -33,7 +34,15 @@ func prepareCockroachDBTestContainer(t *testing.T) (func(), *Config) {
 		if tableName == "" {
 			tableName = defaultTableName
 		}
-		return func() {}, &Config{*s, "vault." + tableName}
+		haTableName := os.Getenv("CR_HA_TABLE")
```

Comment:
I don't think you need Getenv in this test.  I know it's present in some of the existing tests, but it's vestigial mostly.

Reply:
I've removed the `os.Getenv` calls for the `CR_TABLE` and `CR_HA_TABLE` environment variables, so the tests will use the default values.